app/services/saml_request_validator.rb

```diff
@@ -45,6 +39,18 @@ def extra_analytics_attributes
     }
   end
 
+  def parsed_vectors_of_trust
+    return @parsed_vectors_of_trust if defined?(@parsed_vectors_of_trust)
+
+    @parsed_vectors_of_trust = begin
+      if vtr.is_a?(Array) && !vtr.empty?
```

Comment:
The `Array` class check is confusing to me, what happens if it is not an array?
I'd expect something like `Array(vtr).map { ... }` which would seamlessly handle arrays and single items

Reply:
I merged before I saw this comment. I think this could be cleared up with a simple `#blank?` check. We want to fall back to ACR values if it is an empty array or nil. I'll follow up in a new PR